hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/server/FeedListenerGenerator.java

```diff
@@ -101,34 +97,9 @@ public void addListeners(String feedId) throws Exception {
         MsgLogger.LOG.infoAddingListenersForFeed(feedId);
 
         MessageProcessor messageProcessor = new MessageProcessor();
-        String messageSelector = String.format("%s = '%s'", Constants.HEADER_FEEDID, feedId);
```

Comment:
this is how a feed listener only gets messages destined for that feed. this is required - we don't want feeds receiving messages destined for other feeds (especially for queues, since now this means the real feed that should get the message will never get it because the wrong feed got it).